internal/service/organization/resource_organization_migration_test.go

```diff
@@ -26,7 +26,7 @@ func TestMigConfigRSOrganization_Basic(t *testing.T) {
 		Steps: []resource.TestStep{
```

Comment:
curious if there is hard limitation to running these tests in the CI, bug would have been easy to catch if tests ran.

Reply:
The limitation I see is that we need a MongoDB Atlas user to create the orgs. As far as I know we don't have any Atlas account to do this. Currently all tests run using API Keys at the org level.
Maybe it's a good idea to consider creating a team account in Atlas for this kind of tests. WDYT? I can create a ticket for it

Reply:
I see, this would be for providing the org_owner_id right? I would suggest creating a ticket and leave a comment in the skipped test so we have context in the future.

Reply:
Created [CLOUDP-264042](https://jira.mongodb.org/browse/CLOUDP-264042)